expressions/arithmetic.md

```diff
@@ -0,0 +1,84 @@
+# Arithmetic
+
+Pony focusses on two goals, performance and safety. From time to time, these two goals collide. This is true especially for arithmetic on integers and floating point numbers. Safe code should check for overflow, division by zero and other error conditions on each operation where it can happen. Pony tries to enforce as much safety invariants at compile time as it possibly can, but checks on arithmetic operations can only happen at runtime. Performant code should execute integer arithmetic as fast and with as few CPU cycles as possible. Checking for overflow is expensive, doing plain dangerous arithmetic that is possibly subject to overflow is cheap.
+
+Pony provides different ways of doing arithmetic to give the programmer the freedom to chose which operation suits best for him, the safe but slower operation or the fast one, because performance is crucial for the use case.
+
+## Integers
+
+### Ponys default Arithmetic
+
```

Comment:
I take it this is a WIP PR and that content is coming for this section?

Reply:
yes, i just wanted to get some early feedback. If you feel the urge to fill in a section, please go at it, so we get it done sooner. Time is not on my side, unfortunately.

Reply:
definitely not on my side either.

Reply:
content is there